hangman.rb

```diff
@@ -0,0 +1,113 @@
+class Hangman
+
+  def initialize
+    @answer = pick_answer
+    @no_of_guesses = 0
+    @blanks_array = answer_to_blanks
+
+    start
+    puts @answer #temporary!!
+    art
+    get_guess
+
+  end
+
+  def start
+    puts "Welcome to Hangman!"
+  end
+
+  def pick_answer
+    # randomly pick the answer from the library
+    library = ["cat", "dog", "mouse", "letter", "elephant", "bottomless", "house", "polecat", "coffee", "apple"]
+    @answer = library.sample
+
+    return @answer
+  end
+
+  def answer_to_blanks
+    # convert the answer to the number of blanks of the chosen word
+    num_of_blanks = @answer.length
+    @blanks = "_" * num_of_blanks
+    puts @blanks
+    @blanks_array = @blanks.split(//) # turns the string into an array of strings for each character
+
+    return @blanks_array
```

Comment:
Same comment here about the `@blank_array` instance variable. Since you're returning it below, you really should use a local variable here and return it and then set it using an instance variable. Returning it and using the instance variable here is redundant.